src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -2237,9 +2231,9 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
 
         let _group = builder.msg(
             Kind::Test,
-            compiler.stage,
             format!("compiletest suite={suite} mode={mode}"),
-            compiler.host,
+            Mode::ToolBootstrap,
```

Comment:
Remark: I think this isn't quite right, cuz `compiletest` *can* become a `ToolStd` right? But I don't think this is *super* important (it's very niche).

Reply:
Can you just leave a comment? I don't think I care about this being *technically* not always right, since long term we want compiletest to be toolbootstrap anyway.